plugins/tileacq.py

```diff
@@ -369,14 +435,13 @@ def acquire(self, dlg):
                             stream.append(da)
 
                 # Check the FoV is correct using the data, and if not update
-                if i == 0:
-                    afovs = [self._get_fov(d) for d in das]
-                    asfov = (min(f[0] for f in afovs),
-                             min(f[1] for f in afovs))
-                    if not all(util.almost_equal(e, a) for e, a in zip(sfov, asfov)):
-                        logging.warning("Unexpected min FoV = %s, instead of %s", asfov, sfov)
-                        sfov = asfov
-
+#                 if i == 0:
+#                     afovs = [self._get_fov(d) for d in das]
+#                     asfov = (min(f[0] for f in afovs),
+#                              min(f[1] for f in afovs))
+#                     if not all(util.almost_equal(e, a) for e, a in zip(sfov, asfov)):
+#                         logging.warning("Unexpected min FoV = %s, instead of %s", asfov, sfov)
+#                         sfov = asfov
```

Comment:
Is there a way to check the fov of an acquisition stream or does it only work for live streams? I have trouble accessing the pixel size in the _get_fov(sd) function.